common/src/main/res/values/styles.xml

```diff
@@ -245,8 +245,14 @@
         <item name="strokeColor">@color/gray_40</item>
     </style>
 
-    <style name="ExampleBackground" parent="GrayBackground">
-        <item name="cornerRadius">20dp</item>
+    <style name="TaxCategoryExampleBackground">
+        <item name="backgroundColor">@color/tx_category_example_background</item>
+        <item name="strokeColor">@color/tx_category_example_background</item>
```

Comment:
I made some UI fixes to make the tax explainer UI closer to the design, plus added support for the dark theme.